src/backend/controllers/marketController.js

```diff
@@ -151,7 +151,7 @@ exports.addMarketSale = async (req: $Request, res: $Response) => {
                 await sale.save();
 
                 // Send back success confirmation
-                res.status(201).json({ msg: 'Successfully created Market Sale.' });                    
+                res.status(201).json({ msg: 'Successfully created Casus Block Market Sale.' });                    
```

Comment:
hmmm, quite possible we just don't get to this :(

Reply:
Yeah but might as well have it there for when you're able to. Unless that might result in a problem.